.github/workflows/release.yaml

```diff
@@ -0,0 +1,178 @@
+name: Release CSM
+# Invocable as a reusable workflow
+# Can be manually triggered
+on:  # yamllint disable-line rule:truthy
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release (major, minor, patch) Ex: 1.0.0'
```

Comment:
Please change this to `Example` instead of `Ex`

Reply:
Fixed it. Thanks!